opal/Makefile.am

```diff
@@ -22,24 +23,31 @@
 #
 
 if OPAL_cuda_support
-LIBOPAL_CUDA_SUBDIR = cuda
-LIBOPAL_CUDA_LA = cuda/libopalcuda.la
+LIBOPAL_GPU_SUBDIR = cuda
+LIBOPAL_GPU_LA = cuda/libopalcuda.la
+else
+if OPAL_rocm_support
```

Comment:
I am assuming that this comment is moot for the moment, since you mention in later comment on the datatypes that we can in fact only support either CUDA or ROCm at the moment. We can fix this once we have converted to use the accelerator framework.

Reply:
added a check in the main configure.ac file, its just kind of hard to test whether it actually works, since I couldn't find a system with both rocm and cuda installed.